app/code/core/Mage/Core/etc/system.xml

```diff
@@ -62,6 +62,25 @@
                         </use_form_key>
                     </fields>
                 </csrf>
+                <cache>
+                    <label>Advanced Magento cache settings</label>
+                    <frontend_type>text</frontend_type>
+                    <sort_order>10000</sort_order>
+                    <show_in_default>1</show_in_default>
+                    <show_in_website>0</show_in_website>
+                    <show_in_store>0</show_in_store>
+                    <fields>
+                        <flush_cron_expr>
+                            <label>Cron expression for regular cache flushing</label>
+                            <frontend_type>text</frontend_type>
+                            <sort_order>1</sort_order>
+                            <show_in_default>1</show_in_default>
+                            <show_in_website>0</show_in_website>
+                            <show_in_store>0</show_in_store>
+                            <comment>Use 0 0 30 2 0 to never run cron, the recommended setting is 30 3 * * 6</comment>
```

Comment:
Just curious, why do you recommend to run weekly at 03:30 on Saturday instead of the current default of daily at 02:30?

Suggest to enclose the value with double quotes: "0 0 30 2 0", "30 3 * * 6".
```suggestion
                            <comment>Use "0 0 30 2 0" to never run cron, the recommended setting is "30 3 * * 6"</comment>

Reply:
It's mentioned here: https://github.com/OpenMage/magento-lts/pull/2087#discussion_r873553649